doc/sources/conf.py

```diff
@@ -66,7 +66,7 @@
 
 # General substitutions.
 project = 'Kivy'
-copyright = '2010, The Kivy Authors'
+copyright = '2010-2024, 2024, Kivy Team and other contributors'
```

Comment:
The next  release will be performed in 2023, so better to keep 2023 here.

Reply:
@misl6 Done! Thanks.